qiskit/pulse/schedule.py

```diff
@@ -490,7 +490,21 @@ def __init__(self, *schedules, parameters: Optional[Dict[str, Union[float, compl
 
         self._parameterized = tuple(parameterized)
         self._schedules = tuple(full_schedules)
-        self._parameters = tuple(sorted(set(parameters)))
+
+        # pylint: disable=expression-not-assigned
+
+        # NOTE: In IBM Q command definition, FCs in U2, U3 instructions are not ordered
```

Comment:
I find the comment sort of confusing :/ I think the code can speak for itself:
```
        ordered_set = []
        [ordered_set.append(pname) for pname in parameters if pname not in ordered_set]
        self._parameters = tuple(sorted(params_set)) if sort else tuple(params_set)
```
If you change the name of the temporary variable rather than explain it in comments, the intention is quite clear. The three lines together are very straightforward on their own.

The comment seems to be trying to explain why there is even an option to `sort` the parameters. The docstring might be a better place to qualify the meaning or intention of the init args